rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -51,9 +51,9 @@ class Runtime[F[_]: Sync] private (
 
 object Runtime {
 
-  type RhoISpace[F[_]]       = TCPARK[F, ISpace]
-  type RhoPureSpace[F[_]]    = TCPARK[F, PureRSpace]
-  type RhoReplayISpace[F[_]] = TCPARK[F, IReplaySpace]
+  type RhoISpace[F[_]]       = TCPARK0[F, ISpace]
+  type RhoPureSpace[F[_]]    = TCPARK0[F, PureRSpace]
```

Comment:
can we kill pure r space? it's only confusing....
not for this pr

Reply:
Yep. Next PR.